sos/report/plugins/microovn.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2018 Mark Michelson <mmichels@redhat.com>
+# Copyright (C) 2024 Alan Baghumian <alan.baghumian@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This is based on code from ovn_central.py plugin
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import json
+import os
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+class MicroOVN(Plugin, UbuntuPlugin):
+
+    short_desc = 'MicroOVN Snap'
+    plugin_name = "microovn"
+    profiles = ('network', 'virt')
+
+    packages = ('microovn', )
+    commands = ('microovn', 'microovn.ovn-appctl', 'microovn.ovn-nbctl', 'microovn.ovn-sbctl', 'microovn.ovn-trace', 'microovn.ovs-appctl', 'microovn.ovs-dpctl', 'microovn.ovs-ofctl', 'microovn.ovs-vsctl', 'microovn.ovsdb-client', 'microovn.ovsdb-tool',)
+    services = ('snap.microovn.central', 'snap.microovn.daemon', 'snap.microovn.ovn-ovsdb-server-nb', 'snap.microovn.refresh-expiring-certs', 'snap.microovn.switch', 'snap.microovn.chassis', 'snap.microovn.ovn-northd', 'snap.microovn.ovn-ovsdb-server-sb', 'snap.microovn.refresh-expiring-certs.timer',)
+
```

Comment:
flake8 will complain about these lines, so maybe add a couple of commands and a couple of services per line, to help with readability.

Reply:
I did a clean-up and re-committed, hopefully it will get picked up :-D